test/e2e/common/empty_dir.go

```diff
@@ -50,27 +50,27 @@ var _ = ginkgo.Describe("[sig-storage] EmptyDir volumes", func() {
 		})
 
 		ginkgo.It("new files should be created with FSGroup ownership when container is root", func() {
-			doTestSetgidFSGroup(f, testImageRootUid, v1.StorageMediumMemory)
+			doTestSetgidFSGroup(f, nil, v1.StorageMediumMemory)
```

Comment:
why using nil here?

Reply:
That parameter changed from a string (specifying the image name) to to a *int64 (specifying the user as whom to run). ``nil`` besically means that it will run as the image's default user, which maintains the same behaviour.